apps/web/src/db/database.ts

```diff
@@ -0,0 +1,6 @@
+import { buildDatabaseClient } from '@latitude-data/database'
+import env from '$/env'
+
+const testEnv = process.env.NODE_ENV === 'test'
+const connectionString = testEnv ? env.TEST_DATABASE_URL : env.DATABASE_URL
```

Comment:
We let the consumer of the database package provide Db credentials

Reply:
Does it always have to be passed as env variables? Since this is a standalone package, I think this should be passed as a parameter. Then it's the app who decides how to obtain it and, if it's still using an env var, it decides how it's called.

Reply:
agree on it being a parameter

Reply:
It's a parameter. The confusion here is that this file is called `database` but it lives in `apps/web/src/db/database.ts` It use `@latitude/database` init method and it pass the env configuration.